spec/index.html

```diff
@@ -1521,6 +1521,16 @@ <h4>Patterns of RDFS entailment (Informative)</h4>
         </tbody>
       </table>
 
+      <p>As an example of a RDFS entailment involving triple terms using the entailment pattern rdfs14, the following graph:</p>
+
+      <p><code>ex:a ex:b <<( ex:c ex:d <<(ex:e ex:f ex:g)>> )>> .</code></p>
+
+      <p>RDFS entails:</p>
```

Comment:
These emdashes ("&mdash;") are better than the colons, because the colons only tail the "regular body", while the emdashes both lead into and follow after, indicating the continuity all the way into the trailing `code`.
```suggestion
      <p>As an example of a RDFS entailment involving triple terms using the entailment pattern rdfs14, the following graph &emdash;</p>

      <p><code>ex:a ex:b <<( ex:c ex:d <<(ex:e ex:f ex:g)>> )>> .</code></p>

      <p>&emdash; RDFS entails &emdash;</p>
```

Reply:
OK: emdashes are better than the colons.